core/cli/src/main/java/mil/nga/giat/geowave/core/cli/operations/config/ListCommand.java

```diff
@@ -33,6 +38,39 @@
 	public void execute(
 			OperationParams params ) {
 
+		Pair<String, Properties> list = getList(params);
+		String name = list.getKey();
+		Properties p = list.getValue();
+
+		JCommander.getConsole().println(
+				"PROPERTIES (" + name + ")");
+
+		List<String> keys = new ArrayList<String>();
+		keys.addAll(p.stringPropertyNames());
+		Collections.sort(keys);
+
+		for (String key : keys) {
+			String value = (String) p.get(key);
+			JCommander.getConsole().println(
+					key + ": " + value);
+		}
+	}
+
+	@Get("json")
+	public Properties restGet() {
+		filter = getQueryValue("filter");
+
+		OperationParams params = new ManualOperationParams();
+		params.getContext().put(
```

Comment:
Right now it's using the default config. Do we want to be able to include that as a query parameter? If so, it should probably be processed in a centralized place.